### DISCUSSION TITLE
sharing a specific directory from host

### DISCUSSION THREAD

**Discussion by arjver** [lima-vm/lima#1152] [Q&A] [answered]
I see that lima automatically shares `/Users` to the VM, however I would like to only share one directory with the VM (`~/code`, where I keep all my code).

I couldn't find anything about this in the Getting Started or the FAQs... Is this possible? If so, how?

(simply disabling the `/Users` mount is enough, since I can add the `~/code` dir in the yaml)

**Comment by jandubois** [comment, marked as answer]
> (simply disabling the `/Users` mount is enough, since I can add the `~/code` dir in the yaml)

You just modify `lima.yaml` to change the `~` mount to `~/code`. Does that not work for you?

**Comment by arjver** [comment]
Sorry, it seems I just understood the file sharing wrong... I thought lima would share _all_ the files inside my `/Users` folder as read-only, which I did not want.


> You just modify `lima.yaml` to change the `~` mount to `~/code`. Does that not work for you?

That does what I want it to, thanks!
